doc/gallery/plot_rasterio_rgb.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+.. _recipes.rasterio_rgb:
+
+============================
+imshow() and map projections
+============================
+
+Using rasterio's projection information for more accurate plots.
+
+This example extends :ref:`recipes.rasterio` and plots the image in the
+original map projection instead of relying on pcolormesh and a map
+transformation.
+"""
+
+import os
+import urllib.request
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+# Download the file from rasterio's repository
+url = 'https://github.com/mapbox/rasterio/raw/master/tests/data/RGB.byte.tif'
+urllib.request.urlretrieve(url, 'RGB.byte.tif')
+
+# Read the data
+da = xr.open_rasterio('RGB.byte.tif')
+
+# Normalize the image
+da = da / 255
+
+# The data is in UTM projection
+# Until https://github.com/SciTools/cartopy/issues/813 is implemented
+# we have to do this manually
+crs = ccrs.UTM('18N')
+
+# Plot on a map
+ax = plt.subplot(projection=crs)
+da.plot.imshow(ax=ax, transform=crs)
```

Comment:
For pedagogical purposes, it might be nice to explicitly write `rga='band'`.

Reply:
yes, will do